tests/integration/targets/ndo_route_map_policy_route_control_context/tasks/main.yml

```diff
@@ -0,0 +1,566 @@
+        set_rule: {}
+        match_rules: []
+        state: present
+        output_level: debug
+      register: rm_route_map_policy_context_2_values
+
+    - name: Assertion check for update Route Map Policy for Route Control Contexts
+      ansible.builtin.assert:
+        that:
+          - update_route_map_policy_context_2 is changed
+          - update_route_map_policy_context_2.current.action == "permit"
+          - update_route_map_policy_context_2.current.description == "test"
+          - update_route_map_policy_context_2.current.matchRules.0.matchRuleName == "ansible_match_rule_policy_1"
+          - update_route_map_policy_context_2.current.matchRules.0.matchRuleRef is defined
+          - update_route_map_policy_context_2.current.matchRules.0.matchRuleTemplateId is defined
+          - update_route_map_policy_context_2.current.matchRules.0.matchRuleTemplateName == "ansible_tenant_template"
+          - update_route_map_policy_context_2.current.name == "route_map_policy_context_2"
+          - update_route_map_policy_context_2.current.order == 1
+          - update_route_map_policy_context_2.current.setRuleName == "ansible_set_rule_policy_1"
+          - update_route_map_policy_context_2.current.setRuleRef is defined
+          - update_route_map_policy_context_2.current.setRuleTemplateId is defined
+          - update_route_map_policy_context_2.current.setRuleTemplateName == "ansible_tenant_template"
+          - update_route_map_policy_context_2.previous.action == "deny"
+          - update_route_map_policy_context_2.previous.description == "test"
+          - update_route_map_policy_context_2.previous.matchRules.0.matchRuleName == "ansible_match_rule_policy_2"
+          - update_route_map_policy_context_2.previous.matchRules.0.matchRuleRef is defined
+          - update_route_map_policy_context_2.previous.matchRules.0.matchRuleTemplateId is defined
+          - update_route_map_policy_context_2.previous.matchRules.0.matchRuleTemplateName == "ansible_tenant_template"
+          - update_route_map_policy_context_2.previous.name == "route_map_policy_context_2"
+          - update_route_map_policy_context_2.previous.order == 1
+          - update_route_map_policy_context_2.previous.setRuleName == "ansible_set_rule_policy_2"
+          - update_route_map_policy_context_2.previous.setRuleRef is defined
+          - update_route_map_policy_context_2.previous.setRuleTemplateId is defined
+          - update_route_map_policy_context_2.previous.setRuleTemplateName == "ansible_tenant_template"
+          - update_route_map_policy_context_2.proposed.action == "permit"
+          - rm_route_map_policy_context_2_values is changed
+          - rm_route_map_policy_context_2_values.current.action == "permit"
+          - rm_route_map_policy_context_2_values.current.matchRules == []
+          - rm_route_map_policy_context_2_values.current.name == "route_map_policy_context_2"
+          - rm_route_map_policy_context_2_values.current.order == 1
```

Comment:
Looks like some of the removed properties are missed. Should setRuleName/Ref be undefined?
Also should description be undefined if it gets removed with and empty string?

Reply:
Done